src/swsssdk/dbconnector.py

```diff
@@ -17,23 +18,29 @@ def __init__(self, **kwargs):
 
 SonicV1Connector.db_map = _connector_map[SonicV1Connector.__name__]['db_map']
 
-if len(SonicV1Connector.db_map) != len({v['db'] for k, v in SonicV1Connector.db_map.items()}):
-    raise RuntimeError("Duplicate DB index detected in configuration.")
```

Comment:
Why remove this block?

Reply:
> Why remove this block?

I added the missing LOGLEVEL_DB(3) , PFC_WD_DB(5), FLEX_COUNTERS_DB(5) into existing database.json which are mainly used for python today.  Since both PFC_WD_DB and FLEX_COUNTER_EB has value 5. This check will raise error. But these check is unnecessary, internally we are using string "DBNAME" as key to index(in SonicV2Connector/DBInterface), duplicated value index is fine,
 
     "PFC_WD_DB": {
        "db": 5,
        "separator": ":"
      },
      "FLEX_COUNTER_DB": {
        "db": 5,
        "separator": ":"
      },